tests/e2e/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
@kylelaker-ccpo @jasonburkert-ccpo this file was generated by the vie-cli when I add Nightwatch maybe the problems is this is been ignore by github/super-linter@v4